src/components/Navbar/Navbar.tsx

```diff
@@ -4,19 +4,22 @@ import { useState, useEffect } from 'react';
 import Logo from '../../assets/logo-low.png';
 import { useNavigate } from 'react-router-dom';
 import MenuIcon from '@mui/icons-material/Menu';
+import { useAuthContext } from '../../context/AuthContext';
 
 type NavbarProps =  {
-  userName: string | null
+  showAllOptions: boolean
 }
 
-function Navbar({userName}: NavbarProps) {
+function Navbar({showAllOptions}: NavbarProps) {
 
   const [showMenu, setShowMenu] = useState<boolean>(false)
   const [, setIsMobile] = useState<boolean>(false)
   const navigate = useNavigate()
+  const context = useAuthContext();
+  const userName = context.currentUserDataDb!.name
 
   const showMenuStyle = {
-    display: window.innerWidth > 945 ? "flex" : showMenu ? "flex" : "none"
+    display: window.innerWidth > 1000 ? "flex" : showMenu ? "flex" : "none"
   }
```

Comment:
```
  const [showMenu, setShowMenu] = useState<boolean>(false)
  const [, setIsMobile] = useState<boolean>(false)

  const showMenuStyle = {
    display: window.innerWidth > 1000 ? "flex" : showMenu ? "flex" : "none"
  }

  const hamburgerMenuColor = {
    color: showMenu ? "var(--orange)" : "var(--black)"
  }

  const handleHamburgerClick = () => {
    setShowMenu(!showMenu)
  }

  const handleResize = () => {
    if(window.innerWidth <= 1000) {
      setIsMobile(true)
    } else {
      setIsMobile(false)
    }
  }

  useEffect(() => {
    window.addEventListener("resize", handleResize)
  }, [])
  
.......
 <div className={style.hamburgerMenu} onClick={handleHamburgerClick}>
        <MenuIcon style {hamburgerMenuColor}></MenuIcon>
</div>
<div className={style.navbarContainer} style={showMenuStyle}>
.........
```

The logic behind this code is to have the hamburger menu closed when the window is resized <=1000px (where the navbar shows as a hamburger).

When the navbar is more than 1000px, it's always flex. When it's <= 1000px, it can be either flex or none. This depends on the 'showMenu' state, which is changed every time the hamburger menu icon is being pressed.

And yes, I know that the window resize will not cause a react update. That is why I created an event listener in useEffect, which will call "handleResize" function that checks the window size and it will change the isMobileState, which I am using only to create a re-render